### PR TITLE
[Trivial] Modify for code consistency

### PR DESCRIPTION
**Pull request by SeoHyungjun**
The flatten_realizer.cpp and activation_realizer.cpp declare the same local variable. 
However, the actual code order was written differently. 
Minor modifications were made for code consistency.

**Self evaluation:**
1. Build test:   [X]Passed [ ]Failed [ ]Skipped
2. Run test:     [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: SeoHyungjun <hyungjun.seo@samsung.com>

### CODE REVIEW

File: nntrainer/compiler/activation_realizer.cpp [hunk 1]
```diff
@@ -35,6 +35,7 @@ ActivationRealizer::realize(const GraphRepresentation &reference) {
                      std::string /**< layer_name */>
     recovery_table;
   std::vector<LayerNode *> act_nodes;
+
```
Comment (myungjoo) [inline nntrainer/compiler/activation_realizer.cpp:38]: Remove unrelated changes from the commit.

**Review by DonghakPark — APPROVED** [review]
LGTM!

**Review by baek2sm — APPROVED** [review]
LGTM

**Review by songgot — APPROVED** [review]
LGTM
